governance/xc-admin/packages/xc-admin-frontend/utils/capitalizeFirstLetter.ts

```diff
@@ -0,0 +1,3 @@
+export const capitalizeFirstLetter = (str: string) => {
+  return str.replace(/^\w/, (c: string) => c.toUpperCase())
```

Comment:
I don't like having an utils folder tbh. And I feel like you could totally get away without this function. It's only called once.

Reply:
Kind of a nitpick though.

Reply:
It's only called once for now because we are currently only using this for Update Permissions but once we have more tabs we will be using them almost everywhere we need a toast -- what do you suggest other than having an utils folder?